examples/jsm/controls/OrbitControls.js

```diff
@@ -514,6 +604,34 @@ class OrbitControls extends EventDispatcher {
 
 		}
 
+		function updateMouseParameters( event ) {
+
+			if ( ! scope.zoomToCursor ) {
+
+				return;
+
+			}
+
+			performCursorZoom = true;
+
+			const x = event.clientX - scope.domElement.clientLeft;
+			const y = event.clientY - scope.domElement.clientTop;
+			const w = scope.domElement.clientWidth;
+			const h = scope.domElement.clientHeight;
```

Comment:
The cursor position code works only if scope is on the top of browser's screen, because clientLeft is not a suitable property. I suggest the following update:
```suggestion
			const rect = scope.domElement.getBoundingClientRect();
			const x = event.clientX - rect.left;
			const y = event.clientY - rect.top;
			const w = rect.width;
			const h = rect.height;
```

Reply:
Good catch - just pushed a fix with your suggestion.